features/ei-analytics-feature/org.wso2.analytics.solutions.ei.analytics.feature/src/main/resources/siddhi-files/EI_Analytics_StatApp.siddhi

```diff
@@ -53,6 +53,11 @@ define table ESBEventTable (metaTenantId int, messageFlowId string, host string,
 @Index('metaTenantId','entryName')
 define table ConfigEntryTable (metaTenantId int, hashcode string, entryName string, configData string, eventTimestamp long);
 
+-- table that stores component name and type for the searchbox-widget
+@store(type = 'rdbms', datasource = 'EI_ANALYTICS')
+@PrimaryKey('componentName,componentType')
```

Comment:
Do we need both coloumns as primary key?

Reply:
@isharailanga, what is the use case here, do you need keep the unique values in the table?
If that is the case then we dont need to update the table and we can just check the contains and insert(). But if you need to update the records, then the update or insert will be fast then contain check.

Reply:
We shouldnt remove the primary key, however keep only componentType as sprimary key

Reply:
Added componentId column to the table, and made it the primary key in 92ed523702f434f00b5e7a13623cc470edff0b94